tooling/cli/src/dev.rs

```diff
@@ -148,8 +146,6 @@ pub fn setup(options: &mut Options, mobile: bool) -> Result<AppInterface> {
     None
   };
 
-  set_current_dir(tauri_path).with_context(|| "failed to change current working directory")?;
```

Comment:
The `set_current_dir` call was placed here because we need to parse `options.config` relative to the terminal's CWD instead of the tauri directory. It looks weird to run `tauri build --config ../conf.json` and that path be relative to a different directory.